test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -700,4 +711,19 @@ export default class GutenbergEditorComponent extends AbstractEditorComponent {
 		await Promise.all( notices.map( ( notice ) => notice.click() ) );
 		await driverHelper.waitUntilElementNotLocated( this.driver, locator );
 	}
+
+	async dismissNuxWelcomeModal() {
+		const isNuxWelcomeDialogDisplayed = await driverHelper.isElementLocated(
```

Comment:
just noting that we need to be careful to not try to dismiss this if the starter page templates modal is shown (this only happens in the case of a new page), as the welcome tour will be beneath the page templates and the click could be intercepted and cause test failure.  maybe `isElementLocatedAndVisible` may help here as well.

Reply:
also, we probably need this to happen in `site-editor-component` as well, as the same tour can show up there.

Reply:
it may make sense to put this into the `dismiss` method of the guide component, that is imported and used in this file already:

`import GuideComponent from '../components/guide-component.js';`

It looks like that there are two versions of the wpcom welcome guide in ETK - 

https://github.com/Automattic/wp-calypso/blob/5da16704940ad9e01bdd7d9e4d9d1bb4e0863cc1/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/block-editor-nux.js#L55-L66

I THINK we are setting up a dismiss for the DEFAULT_VARIANT, while the guide-component looks like its set up for that old fallback modal variant.

Reply:
Thanks for the review, Addie! I'll move the logic up to the parent, and use `isElementLocatedAndVisible` as you suggested. Like we discussed earlier on Slack, I'll leave any further refactors to follow up PRs and or if we start getting failures related to this somewhere.